nuxeo-tools-hooks/nxtools/hooks/services/jira_service.py

```diff
@@ -61,9 +62,18 @@ def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=N
 class JiraService(AbstractService):
     def __init__(self):
         self.__jira_client = None
+        self.__anonymous_jira_client = None
         if "basic" == self.config("auth_type"):
             self.__jira_client = JiraClient(self.config("url"),
                                             basic_auth=(self.config("basic_username"), self.config("basic_password")))
+            self.__anonymous_jira_client = JiraClient(self.config("url"))
```

Comment:
Maybe a cooment would be interesting for later: the fact that the `.netrc` will be automatically use is not obvious.